### DISCUSSION TITLE
MSSQL Detection?

### DISCUSSION THREAD

**Discussion by zgaspard** [projectdiscovery/nuclei#3374] [Q&A] [answered]
I'm trying to find out if there is a template for detecting MSSQL and I stumbled upon [this](https://github.com/projectdiscovery/nuclei-templates/issues/5415) page, which has MSSQL on port 1433 checked. I may be misinterpreting the page (if so let me know), but this would lean me towards assuming there is a detection. However, I can't seem to find it. Does anyone know if there is a simple MSSQL detection or not? Thanks!

**Comment by olearycrew** [comment, marked as answer]
I actually don't see a template that looks for port 1433 and MSSQL. I've opened this issue: https://github.com/projectdiscovery/nuclei-templates/issues/7064 to track.
